### PR TITLE
[MIG] runbot_skip_test -> 9.0

### PR DESCRIPTION
**Pull request by gurneyalex**
no changes needed

### CODE REVIEW

**Comment by moylop260** [comment]
+1

File: runbot_skip_tests/models/runbot_build.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#    (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.osv import orm
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class runbot_build(orm.Model):
```
Comment (pedrobaeza) [inline runbot_skip_tests/models/runbot_build.py]: Why not new API for making the transition to v10 smoother?

Comment (pedrobaeza) [reply runbot_skip_tests/models/runbot_build.py]: CamelCase for class name.

Comment (moylop260) [reply runbot_skip_tests/models/runbot_build.py]: Thanks!
Fixed

**Comment by jjscarafia** [comment]
:+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**